docs/api.rst

```diff
@@ -44,6 +44,7 @@ Preferential Optimization
    optuna_dashboard.preferential.create_study
    optuna_dashboard.preferential.load_study
    optuna_dashboard.preferential.PreferentialStudy
+   optuna_dashboard._preference_setting.register_preference_feedback_component
```

Comment:
You need to export from the public module. Could you update the `__init__.py` as well?

```suggestion
   optuna_dashboard.register_preference_feedback_component
```